docker-compose.yml

```diff
@@ -0,0 +1,59 @@
+version: "3.7"
+services:
+  radicle-node:
+    image: gcr.io/radicle-services/radicle-node:${RADICLE_IMAGE_TAG:-latest}
+    build:
+      dockerfile: ./radicle-node/Dockerfile
+      context: .
+    environment:
+      RUST_LOG: hyper=warn,debug
```

Comment:
We can just set this to `debug` as there's no hyper in the node.

Reply:
Gotcha 👍